frontend/packages/console-plugin-shared/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@console/plugin-shared",
```

we will be publishing under openshift-console npm org
```suggestion
  "name": "@openshift-console/plugin-shared",
```

---

done